docs/articles/expensify-classic/workspace-and-domain-settings/Categories.md

```diff
@@ -1,5 +1,137 @@
+
+- CSV
+- TXT 
+- XLS 
+- XLSX
+  
+Once the spreadsheet is formatted, you can upload it to the workspace under Settings > Workspace > Group >  Workspace Name > Categories. 
+From there, the updated Category list will show as available on all expenses submitted on the corresponding workspace. 
+## Manually add Categories
+If you need to add Categories to your workspace manually, you can follow the steps below. 
+
+On web: 
+1. Navigate to Settings > Workspace > Group / Individual > [Workspace name] > Categories. 
+2. Add new categories under Add a Category.
+   
+On mobile:
+1. Tap the three-bar menu icon at the top left corner of the app
+2. Tap on Settings in the menu on the left side
+3. Scroll to the Workspace subhead and click on Categories listed underneath the default policy
+4. Add new categories by tapping the + button in the upper right corner. To delete a category, on iOS swipe left, on Android press and hold. Tap a category name to edit it.
+## Add sub-categories
+Sub-categories are useful when you want to be more specific, i.e. Travel could have a subcategory of airplane or lodging so the outcome would be Travel:airplane or Travel:lodging.
+If you would like to create sub-categories under your category selection drop-down list, you can do so by adding a colon after the name of the desired category and then typing the sub-category (without spaces around the punctuation).
+# How to import Categories with an accounting integration connected
+If you connect Expensify to a direct integration such as QuickBooks Online, QuickBooks Desktop, Sage Intacct, Xero, or NetSuite, then Expensify automatically imports the general ledger parameters from your accounting system as Categories.
+
+When you first connect your accounting integration your categories will most likely be pulled from your chart of accounts, however this can vary depending on the account integration. 
+
+If you need to update your categories in Expensify, you will first need to update them in your accounting system, then sync the connection in Expensify by navigating to Settings > Workspace > Group > [Workspace Name] > Connection > Sync Now.
+
+Alternatively, if you update the category details in your accounting integration, be sure to sync the policy connection so that the updated information is available on the workspace. 
+# Deep Dive
+## Category-specific rules and description hints
+If you're an admin using a workspace on a Control plan, you have the ability to enable specific rules for each category.
+
+These settings are valuable if you want to set a special limit for a certain category. e.g. Your default expense limit is $2500 but Entertainment has a limit of $150 per person, per day. You can also require or not require receipts, which is great for allowing things like Mileage or Per Diems to have no receipt.
+
+To set up Category Rules, go to Settings > Workspace> Group > [Workspace Name] > Categories.
```

Comment:
Missing formatting:
**Settings > Workspace> Group > _[Workspace Name]_ > Categories**

Reply:
_[Workspace Name]_ should be in italic.

Reply:
I resolved this